### PR TITLE
[New Rule] Potential Pass The Hash

### PR DESCRIPTION
**Pull request by Samirbous**
Using new term rule type plus PTH pattern using windows log 4624.

https://github.com/elastic/detection-rules/issues/1710

image

### CODE REVIEW

**Review by terrancedejesus — APPROVED** [review]
LGTM. Discussed with @Samirbous about the use of secondary login and new terms to reduce FPs since the rule will detect a user logging in with new credentials. Great turnaround time 😄

File: rules/windows/lateral_movement_alternate_creds_pth.toml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may pass the hash using stolen password hashes to move laterally within an environment, 
+bypassing normal system access controls. Pass the hash (PtH) is a method of authenticating as a user 
+without having access to the user's cleartext password. 
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
```
Comment (w0rk3r) [inline rules/windows/lateral_movement_alternate_creds_pth.toml:17]:
```suggestion
index = ["winlogbeat-*", "logs-windows.*", "logs-system.*"]
```
I think that auth logs are collected using the System one, except for WEF

File: rules/windows/lateral_movement_alternate_creds_pth.toml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2023/03/29"
+integration = ["windows", "endpoint"]
```
Comment (w0rk3r) [inline rules/windows/lateral_movement_alternate_creds_pth.toml:3]:
```suggestion
integration = ["windows", "system"]
```

File: rules/windows/lateral_movement_alternate_creds_pth.toml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+description = """
+Adversaries may pass the hash using stolen password hashes to move laterally within an environment, 
+bypassing normal system access controls. Pass the hash (PtH) is a method of authenticating as a user 
+without having access to the user's cleartext password. 
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Potential Pass the Hash Attempt"
```
Comment (w0rk3r) [inline rules/windows/lateral_movement_alternate_creds_pth.toml:20]:
```suggestion
name = "Potential Pass-the-Hash (PtH) Attempt"
```
This one is optional, just to facilitate searchs

**Review by w0rk3r — APPROVED** [review]
🧙🏼‍♂️

**Pull request merged**